lib/utils/is-tar-gz.js

```diff
@@ -0,0 +1,5 @@
+var regex = /((\.tar(\.gz)?)|(\.tgz))$/;
```

Comment:
Can be `const` instead of `var`

Reply:
I changed to `const`, I just used `var` because the whole repo uses `var`